src/fluree/server/system.clj

```diff
@@ -126,24 +134,70 @@
 
 (def default-resource-name "file-config.jsonld")
 
+(defn log-config-summary
+  [parsed-config]
+  (let [connection-config (some #(when (conn-config/type? % conn-vocab/connection-type) %)
+                                (vals parsed-config))
+        storage-config (some #(when (conn-config/type? % conn-vocab/storage-type) %)
+                             (vals parsed-config))
+        consensus-config (some #(when (config/consensus? %) %)
+                               (vals parsed-config))
+        http-config (some #(when (config/http-api? %) %)
+                          (vals parsed-config))
+        cache-mb (when connection-config
+                   (conn-config/get-first-integer connection-config conn-vocab/cache-max-mb))
+        storage-path (when storage-config
+                       (conn-config/get-first-string storage-config conn-vocab/file-path))
+        consensus-type (when consensus-config
+                         (util/get-first-value consensus-config server-vocab/consensus-protocol))
+        max-pending-txns (when (and consensus-config (= "standalone" consensus-type))
+                           (conn-config/get-first-integer consensus-config server-vocab/max-pending-txns))
+        http-port (when http-config
+                    (conn-config/get-first-integer http-config server-vocab/http-port))
+        closed-mode (when http-config
+                      (conn-config/get-first-boolean http-config server-vocab/closed-mode))]
+    (log/info "Server configuration summary:")
+    (log/info "  HTTP port:" (or http-port "Not configured"))
+    (log/info "  Consensus mode:" (or consensus-type "Not configured"))
+    (when max-pending-txns
+      (log/info "  Max pending transactions:" max-pending-txns))
+    (log/info "  Storage:" (if storage-path
+                             (str "File-based at " storage-path)
+                             "In-memory"))
```

Comment:
We theoretically support more storage types than file and memory like s3 and ipfs for example. Whether the other ones actually work or not is another question.